besu/src/main/java/org/web3j/tx/PrivateTransactionManager.java

```diff
@@ -179,34 +179,34 @@ public EthSendTransaction sendEIP1559Transaction(
             boolean constructor)
             throws IOException {
         final BigInteger nonce =
-                besu.privGetTransactionCount(txSignService.getAddress(), privacyGroupId)
+                besu.privGetTransactionCount(privateTxSignService.getAddress(), privacyGroupId)
                         .send()
                         .getTransactionCount();
 
-        final RawPrivateTransaction transaction;
+        final PrivateTransaction1559 transaction;
         if (privateFor != null) {
             transaction =
-                    RawPrivateTransaction.createTransaction(
+                    new PrivateTransaction1559(
```

Comment:
It seem some places the pattern is LegacyPrivateTransaction.createTransaction(..) and here you're using a different pattern by calling the constructor directly on PrivateTransaction1559. Might be better to use the same pattern throughout, and if you go with the createTransaction(..) apporach, could you be able to support private 1559 transactions without breaking any existing APIs?

Reply:
Moved to .createTransaction(..) pattern everywhere